### PR TITLE
Disable artefact uploading

### PR DESCRIPTION
**Pull request by f-squirrel**

### CODE REVIEW

File: tests/apollo/test_skvbc_view_change.py [hunk 1]
```diff
@@ -533,6 +533,7 @@ async def _wait_for_replica_to_ask_to_leave_view_due_to_status(self, bft_network
             while True:
                 ask_to_leave_msg_count = await bft_network.get_metric(
                     node, bft_network, "Counters", "sentReplicaAsksToLeaveViewMsgDueToStatus")
+                ask_to_leave_msg_count = 0
```
Comment (teoparvanov) [inline tests/apollo/test_skvbc_view_change.py:536]: I suppose the goal of this small change is to trigger a failure?

**Review by teoparvanov — APPROVED** [review]
LGTM!

File: tests/apollo/test_skvbc_view_change.py (hunk 1 above)
Comment (f-squirrel) [reply tests/apollo/test_skvbc_view_change.py:536]: @teoparvanov , yes, I have mentioned it in the commit message "Commit to check failed logs"

Comment (teoparvanov) [reply tests/apollo/test_skvbc_view_change.py:536]: Thanks for clarifying!

**Pull request merged**